lib/App/DuckPAN/Fathead.pm

```diff
@@ -105,10 +105,10 @@ sub _search_output {
 
 	my $result = $self->_db_lookup($query);
 
-	if ($result && $result->{type} eq "R") {
+	while ($result && $result->{type} eq 'R') {
```

Comment:
@GuiltyDolphin I don't expect this will ever run more than once, but thanks for fixing the bug here!